README.md

```diff
@@ -403,6 +404,88 @@ use Omnimail\Omnimail;
 $mailer = Omnimail::create(Omnimail::AMAZONSES, ['accessKey' => $accessKey, 'secretKey' => $secretKey]);
 ```
 
+<a name="mass-mailings"></a>
+## Mass Mailings
+
+The mass mailing component is for interacting with mass mailing providers. Currently the code focusses on data retrieval, but in future it should also define creating and sending mass mailings.
+
+There are 2 functions currently described for the Mass mailings interface `getMailings` and `getRecipients`.
+
+__getMailings__
+
+```
+    $mailer = Omnimail::create('Silverpop')->getMailings();
+    $mailer->setUserName($userName);
+    $mailer->setPassword($password);
+    $mailer->setStartTimeStamp(strtotime('7 days ago'));
+    $mailer->setEndTimeStamp(strtotime('now'));
+    // Instead of using set methods an array can be passed into create -ie 
+    // Omnimail::create('Silverpop', array('UserName' => $userName...);
+
+    $mailings = $mailer->getResponse();
+    for ($i = 0; $i < 15; $i++) {
+      if (!$mailings->isCompleted()) {
```

Comment:
Yeah in some cases (e.g millions on entries) that isn't actually long enough - in that case 'retrieval parameters' are returned & they need to be re-passed in when recalling.